client/src/pages/routes/route.ts

```diff
@@ -67,43 +66,15 @@ const menuItems = [
 
 export const getAllRoutes = () => menuItems;
 
-// export const getPetSitterRoutes = () => {
-//   const result = [];
-//   for (const item of menuItems) {
-//     if (item.canView?.length === 2) {
-//       result.push(item);
-//       continue;
-//     } else {
-//       if (item.canView[0] === 'pet_sitter') {
-//         result.push(item);
-//         continue;
-//       }
-//     }
-//   }
-//   return result;
-// };
-
-// export const getPetOwnerRoutes = () => {
-//   const result = [];
-//   for (const item of menuItems) {
-//     if (item.canView?.length === 2) {
-//       result.push(item);
-//       continue;
-//     } else {
-//       if (item.canView[0] === 'pet_owner') {
-//         result.push(item);
-//         continue;
-//       }
-//     }
-//   }
-//   return result;
-// };
-
-export const getNonAuthenticatedRoutes = () => {
-  const result = [];
-
-  for (const item of menuItems) {
-    if (!item.authenticated) result.push(item);
+export const getRoutesAccordingToAccountType = (accountType: string) => {
+  const routes = [];
+  for (const route of menuItems) {
+    if (route.canView?.length > 1) {
+      routes.push(route);
```

Comment:
Logically this doesn't make a ton of sense. If the canView is for example `[AccountType.SUPER_ADMIN, AccountType.OWNER]` then `AccountType.ADMIN` would be able to view it. This isn't very secure. We should only be rendering the route based if their account type is included in the routes. `Array.includes` is the method.